### DISCUSSION TITLE
How to get "Input" props

### DISCUSSION THREAD

**Discussion by erwinstone** [mantinedev/mantine#3712] [Q&A] [answered]
I usually use "React.ComponentProps" to get the props type of a component. Is there another way to get the props from the Input component

![image](https://user-images.githubusercontent.com/96282452/223649608-47a43fb7-026e-46ca-ab86-3a4a8e6b2587.png)

**Comment by rtivital** [comment, marked as answer]
`import { InputProps } from '@mantine/core'`

**Reply by erwinstone** [reply]
But there is no "value" prop

https://user-images.githubusercontent.com/96282452/223650745-4f8dd500-fd82-4f01-a42f-188e5fa7f7f5.mp4

**Reply by rtivital** [reply]
`InputProps & React.ComponentPropsWithoutRef<'input'>`

**Reply by erwinstone** [reply]
Amazing, Thank you
